Fix regression #6864 caused by #6613

pkowal1982 opened the pull request:
Should fix the issue. Calling remove should delete all matching tweens but internally it stops on first match so should work with delayed tweens as before.

bojidar-bg commented on scene/animation/tween.cpp:
```diff
@@ -620,7 +620,7 @@ void Tween::_tween_process(float p_delta) {
 				}
 				if (!repeat)
-					call_deferred("remove", object, data.key);
+					call_deferred("_remove", object, data.key, true);
```
You never bind `_remove`, so this would likely not work (+ spam the console).

pkowal1982 commented:
You're right @bojidar-bg. I think exposing two almost the same methods to the user has little sense and is not elegant. I've just added additional default param to `remove()` method.
It's not consistent with three other but now it should work exactly as in 2.1 and will also enable user to remove tweens in packs.
Also updated documentation.
I think I will try find more elegant solution for 3.0.
Is there a way to bind method but not expose it to the user?

bojidar-bg commented:
@pkowal1982 Well, starting it with `_` wouldn't expose it to the user, though they would still be able to call it...

Commit:
Fix regression #6864 caused by #6613

pkowal1982 commented:
OK, so now it looks consistent with other methods and _remove is bound but still not visible to user. Thanks @bojidar-bg for help and review.
